Dockerfile

```
@@ -12,22 +12,12 @@ LABEL maintainer="Citus Data https://citusdata.com" \
       org.label-schema.schema-version="1.0"
 
 ENV CITUS_VERSION ${VERSION}.citus-1
-#install postgresql
-RUN apt-get update && apt-get install -y lsb-release apt-utils wget curl gnupg2 && apt-get clean all \
-    && wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
-    && sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main 14" > /etc/apt/sources.list.d/pgdg.list' \
-    && apt-get purge -y --auto-remove curl \
-    && rm -rf /var/lib/apt/lists/*
 
 # install Citus
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
        ca-certificates \
-       curl \
-    && curl -s https://install.citusdata.com/community/deb.sh |  bash \
-    &&  apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
-    && apt-get purge -y --auto-remove curl \
-    && rm -rf /var/lib/apt/lists/*
+    && curl -s https://install.citusdata.com/community/deb.sh | bash
```

```suggestion
    curl \
    && curl -s https://install.citusdata.com/community/deb.sh | bash
    &&  apt-get install -y postgresql-$PG_MAJOR-citus-10.2=$CITUS_VERSION \
    && apt-get purge -y --auto-remove curl \
    && rm -rf /var/lib/apt/lists/*
```

You are missing some steps in here